TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Shared/_CheckboxFilter.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@model CheckboxFilterViewModel
+
+<govuk-checkboxes id-prefix="trs-filters-@Model.Name" name=@Model.Name checkboxes-class="govuk-checkboxes govuk-checkboxes--small">
```

Comment:
`govuk-checkboxes` should be applied by default
```suggestion
<govuk-checkboxes id-prefix="trs-filters-@Model.Name" name=@Model.Name checkboxes-class="govuk-checkboxes--small">
```

Reply:
Fixed